Documentation~/index.md

```diff
@@ -21,7 +21,7 @@ To install this package, refer to the instructions that match your Unity Editor
 
 ### Version 2021.1 and later
 
-To install this package, follow the instructions for [adding a package by name](https://docs.unity3d.com/2021.1/Documentation/Manual/upm-ui-quick.html) in the Unity Editor. 
+To install this package, follow the instructions for [adding a package by name](https://docs.unity3d.com/2021.1/Documentation/Manual/upm-ui-quick.html) in the Unity Editor. The package's name is com.unity.project-auditor .
```

Comment:
what do you think about formatting com.unity.project-auditor differently instead of adding an extra space before the fullstop?